### DISCUSSION TITLE
Text not lining up?

### DISCUSSION THREAD

**Discussion by Ren-B-7** [goolord/alpha-nvim#309] [Q&A] [answered]
Header is set to centre, but it defaults my column count to 80, and thus doesn't count columns properly?
Strangely it can centre groups but not text items

![image](https://github.com/user-attachments/assets/fa92eb94-b84d-4ab1-80a6-3dd16c217748)

I dont set opt.columns in my options, and the terminal size doesnt affect this, a terminal smaller than 80 columns also defaults to 80?

[Screencast from 2024-10-24 02-57-34.webm](https://github.com/user-attachments/assets/d6318f07-d38b-4fc9-840d-86b1a5ad4f25)

Its not a setting that i am making and not an issue in my [source](https://github.com/Ren-B-7/Super-Start.nvim/blob/4f00237992385de065ae7662ac9715e9ae4470c7/nvim/lua/plugins/alpha_intro.lua)

**Comment by goolord** [comment, marked as answer]
https://github.com/Ren-B-7/Super-Start.nvim/blob/4f00237992385de065ae7662ac9715e9ae4470c7/nvim/lua/plugins/alpha_intro.lua#L26

you are using the British spelling "centre", while the accepted value is "center"

**Comment by Ren-B-7** [comment]
Thats just depressing
